packages/shared/src/variants/baduk.ts

```diff
@@ -190,19 +216,48 @@ export class Baduk extends AbstractGame<BadukConfig, BadukState> {
     this.phase = "gameover";
   }
 
-  defaultConfig(): BadukConfig {
+  defaultConfig(): GridBadukConfig {
     return { width: 19, height: 19, komi: 6.5 };
```

Comment:
If top level `width` and `height` are now considered legacy, we may want to update this return value.

Reply:
changed to return new type structure